test/scripts/boot-image

```diff
@@ -304,6 +305,61 @@ def _boot_qemu_iso(arch, installer_iso_path, config_file, privkey_path):
                 qemu_cmd_scp_and_run(vm, cmd, privkey_path)
 
 
+def boot_qemu_pxe(arch, pxe_tar_path, config_file):
+    with contextlib.ExitStack() as cm:
+        # unpack the tar and create a combined image
+        tmpdir = cm.enter_context(TemporaryDirectory(dir="/var/tmp"))
+        root_dir = pathlib.Path(tmpdir)
+        subprocess.check_call(
+            ["tar", "-C", root_dir, "-x", "-f", pxe_tar_path])
```

Comment:
We use `tmpdir` in the other functions and suddenly `root_dir` here even though they refer to the same path. I understand why you convert it to a `pathlib.Path` (it's easier later on) but why not use `root_dir` everywhere, or `tmpdir` everywhere right now it flip-flops between the two (`root_dir` here, `tmpdir` on the next two lines `root_dir` on line 325, etc).

As a nitpick: `root_dir` vs `tmpdir` how about `rootdir`, `tmpdir` or `root_dir`, `tmp_dir`? ;)